deploy/deploy.sh

```diff
@@ -62,51 +62,41 @@ declare -a application_list=(
 )
 
 create() {
-    # create our objects by building a list
-    built_list=""
     object_list=("$@")
-    for spec in ${object_list[@]}; do
-        built_list+=" -f $spec"
-    done
-
-    oc create --save-config=true $built_list
+    # shellcheck disable=SC2068
```

Comment:
The existing code already failed SC2068 - the only alternative here is to put the `-f` on every line of our array, which seems silly.